x/commercioauth/msgs_test.go

```diff
@@ -0,0 +1,140 @@
+package commercioauth
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	"testing"
+)
+
+func (this MsgCreateAccount) isEqualTo(msg MsgCreateAccount) bool {
+
+	if this.Signer.String() == msg.Signer.String() {
+		if this.Address == msg.Address {
+			if this.KeyType == msg.KeyType {
+				if this.KeyValue == msg.KeyValue {
+					return true
+				}
+			}
+		}
+	}
```

Comment:
Can we simplify this statement removing all the `if`s and replacing them with `&&`? 

Such as 

```go
return this.Signer.String() == msg.Signer.String() && 
   this.Address == msg.Address && 
   this.KeyType == msg.KeyType && 
   this.KeyValue == msg.KeyValue
```
```